src/modules/site-v2/base/static/js/plot.js

```diff
@@ -6,6 +6,14 @@
 
 
 
+/* Enum type mapping plot dimension names to corresponding indices. */
+const PlotDimension = {
+  'x_axis': 0,
```

Comment:
the values `0` and `1` might have special meaning in the current context. For other use cases, consider `Symbol('name');`